.development.env

```diff
@@ -0,0 +1,7 @@
+PORT='9393'
```

Comment:
I would vote we just always run the app on this port. We can tweak the heroku settings but I can't think of any reason we really need to be switching between ports based on environments